content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -158,6 +158,7 @@ different Kubernetes components.
 | `PodDeletionCost` | `false` | Alpha | 1.21 | 1.21 |
 | `PodDeletionCost` | `true` | Beta | 1.22 | |
 | `PodHasNetworkCondition` | `false` | Alpha | 1.25 | |
+| `PodSchedulingReadiness` | `false` | Alpha | 1.26 | |
```

Comment:
Please add a description for this gate (at the end of this page) as we do for other features.

Reply:
will do. For now it's a placeholder.